.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ base_job: &base_job
   executor:
     name: android/android-machine
     resource-class: xlarge
-    tag: 2023.11.1 #https://circleci.com/developer/images/image/cimg/android
```

Comment:
Copying over Alex's change in master to fix CI: https://github.com/kickstarter/android-oss/pull/1985

Reply:
super nitpick comment here, could we bring back the comment or add some kind of documentation in guru ?
There will be a moment when testing for new operative system, (Android 15 in this year) when we will need to do spikes, and likely `default` won't contain Android 15 SDK nor all the software releated and we will have to use a tag